greek/lighter/lighter.md

```diff
@@ -0,0 +1,5 @@
+Ο αναπτήρας δε δουλεύει. Σκατά. Το ζαχαρωτό σε κοροιδεύει διότι αγοράζεις μόνο BIC αναπτήρες που ή δε δουλεύουν ή στους καβατζάρουν οι φίλοι.
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.